docs/integration/kafka.md

```diff
@@ -1,8 +1,181 @@
 title: Kafka Monitoring Integration
 description: Monitor Kafka metrics for brokers, producers, and consumers, consumer lag and offset monitoring by consumer group, topic, or partition, and more. Our cloud and on-premises tools provide out of box Kafka graphs, reports and custom dashboards with built-in anomaly detection, threshold, and heartbeat alerts as well as easy chatops integrations
 
+
+Sematext has a simple Kafka monitoring Agent written in Java and Go with minimal CPU and memory overhead. It's easy to install and doesn't require any changes to the Kafka source code or your application's source code.
+
+## Sematext Kafka Monitoring Agent 
+This lightweight, open-source [Java Monitoring Agent](https://github.com/sematext/sematext-agent-java) collects Kafka performance metrics and sends them to Sematext. It comes packaged with a Golang-based agent responsible for Operating System level metrics like network, disk I/O, and more. The Kafka Monitoring Agent can be installed with RPM/DEB package manager on any host running Linux or in a containerized environment using ```sematext/spm-client```.
```

Comment:
Hey @gr0 @adnanrahic @bsmid when you read this "....Java Monitoring Agent" I always get this bad feeling that this sounds like it's an agent for monitoring Java/JVM apps.  The language of implementation is not really important.  We only use it do differentiate between agents written in different languages, but that is not something users (need to) care/think about.  For example, in this case, we wouldn't lose anything by just saying "This lightweight, open-source Monitoring Agent....". Plus, look how we later refer to it as "Kafka Monitoring Agent", which is really more relevant.  It's more important to call it by what it's monitoring than what language it is written in.

Reply:
Do you think that changing the `Java Monitoring Agent` to `Kafka Monitoring Agent` sounds better to `Monitoring Agent` itself? I like the `Kafka Monitoring Agent` better, but I'm not sure.

Reply:
👍 for <integration> Monitoring Agent

Reply:
Agree. Just using **Monitoring Agent** is optimal here.